2019-Feburary-26/writingSamples/tong.tex

```diff
@@ -0,0 +1,165 @@
+level.
+
+These limitations can be addressed with machine learning, which is a
+multivariate method to detect trends and patterns in existing data,
+which can later be used to make predictions on new data. Machine
+learning also allows for prediction to be made at the individual level.
+
+The gap in the literature that this project hopes to address is the
+inconsistent findings in brain structure and functional connectivity in
+FH of AUD. Using a large sample from the ABCD database is one of the
+strategies to achieve reliable results. Using machine learning is the
+additional strategy to improve reliability, as machine learning can
+extract replicable brain patterns that distinguish FHP from FHN. In
+particular, machine learning algorithms can pick up the overall patterns
+of a dataset while ignoring noisy, idiosyncratic characteristics that
+are specific to that dataset, increasing generalizability and
+replicability of the results.
+
+\textbf{3.2.1 Strengths of deep learning}
+
+One criticism of conventional machine learning techniques is not
+performing well on raw data, especially high dimensional brain imaging
+data, necessitating the ``feature selection'' step, which involves
+discarding redundant information to focus solely on informative
+features. Feature selection is oftentimes subjective and requires
+extensive prior knowledge deducing what features should be considered
+informative.
+
+Deep learning, on the other hand, is more objective as it can learn
+features from raw data, bypassing the ``feature selection'' step.
+Inconsistent findings from previous studies on FH of AUD demonstrated
+the subtle and diffuse structural and connectivity abnormalities
+associated with family history status. Deep learning can outperform
+traditional machine learning in datasets with such subtle and diffuse
+patterns (LeCun et al., 2015). In addition, unlike univariate methods,
+deep learning automatically takes into account the inter-correlation
+between features and down-weight unmeaningful interactions.
+
+Indeed, several studies have demonstrated deep learning's superior
+performance relative to other machine learning methods (Koyamada et al.
+2015; Plis et al. 2014). For instance, using Deep Neural Networks to
+classify brain states associated 7 different tasks (Emotion, Gambling,
+Language, Motor, Relational, Social and Working Memory) from 499
+subjects, Koyamada et al. (2015) found better performance for a Deep
+Neural Networks model (mean accuracy of 50.74\%) compared to supervised
+learning methods (mean accuracy of 47.97\%) such as Linear Regression
+and Support Vector Machine. In a different study, using a Deep Belief
+Network model to classify T1-weighted structural data from 198
+schizophrenic patients and 191 controls, Plis et al. (2014) achieved
+90\% classification accuracy with deep learning, while a Support Vector
+Machine model only achieved 68\% classification accuracy.
```

Comment:
connect back to your study at the end and say that deep learning may hold promise with this data.